src/main/java/com/iridium/iridiumteams/gui/BoostersGUI.java

```diff
@@ -56,15 +56,24 @@ public void addContent(Inventory inventory) {
             int currentLevel = teamEnhancement.isActive(enhancementEntry.getValue().type) ? teamEnhancement.getLevel() : 0;
             String nextLevel = nextData == null ? iridiumTeams.getMessages().nullPlaceholder : String.valueOf(currentLevel + 1);
             String cost = nextData == null ? currentData == null ? iridiumTeams.getMessages().nullPlaceholder : String.valueOf(currentData.money) : String.valueOf(nextData.money);
+            String minLevel = nextData == null ? iridiumTeams.getMessages().nullPlaceholder : String.valueOf(nextData.minLevel);
             List<Placeholder> placeholders = currentData == null ? new ArrayList<>() : new ArrayList<>(currentData.getPlaceholders());
             placeholders.addAll(Arrays.asList(
                     new Placeholder("timeremaining_hours", String.valueOf(hours)),
                     new Placeholder("timeremaining_minutes", String.valueOf(minutes)),
                     new Placeholder("timeremaining_seconds", String.valueOf(seconds)),
                     new Placeholder("current_level", String.valueOf(currentLevel)),
+                    new Placeholder("minLevel", minLevel),
                     new Placeholder("next_level", nextLevel),
-                    new Placeholder("cost", cost)
+                    new Placeholder("vault_cost", cost)
```

Comment:
do we also need to add the %cost% placeholder here for legacy reasons (otherwise old configs will break)

Reply:
old configs wont "break", they just won't show the cost correctly, which is an easy config fix (we can still fix this to include that if you'd want, i just renamed it for consistency)